controllers/k8ssandracluster_controller.go

```diff
@@ -434,9 +456,21 @@ func (r *K8ssandraClusterReconciler) setStatusForStargate(kc *api.K8ssandraClust
 		}
 	}
 
+	if kc.Status.Datacenters[dcName].Stargate.Progress == "" {
```

Comment:
This is because the empty string is not a valid Stargate status and causes the k8ssandracluster controller to spit errors to the logs.